### PR TITLE
Reviews react container

### PR DESCRIPTION
**Pull request by alexvis**
Reviews component and container ready with rspec

### CODE REVIEW

File: react/src/containers/ParkShow.js [hunk 1]
```diff
@@ -30,20 +31,28 @@ class ParkShow extends React.Component {
 
   render() {
     let ratings
+    let park_name
```
Comment (chrissyfancy) [inline react/src/containers/ParkShow.js]: We use camel case when defining React variables. Please change all instances of `park_name` to `parkName`

File: react/test/components/ReviewTileSpec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      wrapper = mount(
+        <ReviewTile
+          reviewContent = {review}
+        />
+      )
+    });
+
+  it('should return true', () => {
+    expect(true).toEqual(true)
+  })
```
Comment (chrissyfancy) [inline react/test/components/ReviewTileSpec.js]: You can remove this test now.

**Comment by chrissyfancy** [comment]
Please address the comments I made, and then I will merge it in!

File: react/test/components/ReviewTileSpec.js (hunk 2 above)
Comment (JesseChamberlain) [reply react/test/components/ReviewTileSpec.js]: - corrected

File: react/src/containers/ParkShow.js (hunk 1 above)
Comment (JesseChamberlain) [reply react/src/containers/ParkShow.js]: - corrected

**Comment by chrissyfancy** [comment]
Just looked through and the codeclimate errors are fine. I will merge now!

**Pull request merged**